packages/twenty-server/src/engine/metadata-modules/data-source/data-source.entity.ts

```diff
@@ -1,18 +1,20 @@
 import {
   Column,
   CreateDateColumn,
+  DataSourceOptions,
   Entity,
+  Index,
+  OneToMany,
   PrimaryGeneratedColumn,
   UpdateDateColumn,
-  DataSourceOptions,
-  OneToMany,
 } from 'typeorm';
 
 import { ObjectMetadataEntity } from 'src/engine/metadata-modules/object-metadata/object-metadata.entity';
 
 export type DataSourceType = DataSourceOptions['type'];
 
 @Entity('dataSource')
+@Index('IDX_DATA_SOURCE_WORKSPACE_ID_CREATED_AT', ['workspaceId', 'createdAt'])
```

Comment:
discovered that this index was missing (manually added in production but in wrong order)